component/_forms.scss

```diff
@@ -31,11 +32,18 @@ label {}
 		font-style: italic;
 	}
 
-	@include touch-hover() {
+  // Hint the user that this element is interactive
+  &:hover {
 		border-color: darken($form-input-border-focus, 10%);
 		outline: none;
 	}
 
+  // Adjust colors when this element is focused
+	&:focus {
```

Comment:
indentation is off.

Another note, there is inconsistency with tabs and spaces throughout the different files.